bin/git-mr

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if test "$1" = "clean"; then
+  git for-each-ref refs/heads/mr/* --format='%(refname)' | while read ref; do
+    git branch -D ${ref#refs/heads/}
+  done
+  exit 0
+elif [[ $1 =~ ^(https?://[^/]+/(.+))/merge_requests/([0-9]+).*$ ]]; then
+  remote=${BASH_REMATCH[1]}.git
+  id=${BASH_REMATCH[3]}
+  branch=mr/$id
+else
+  test -z $1 && echo "mr number required." 1>&2 && exit 1
```

Comment:
If I run `git mr`, I got
```
/usr/local/bin/git-mr: line 28: $1: unbound variable
```
But when  I run `git pr`, I got:
```
pr number required.
```

IMHO, I hope we could keep the consistency of error message.